src/parser/mod.rs

```diff
@@ -2231,6 +2231,18 @@ impl<'a> Parser<'a> {
         ))
     }
 
+    fn parse_duckdb_struct_type_def(&mut self) -> Result<Vec<StructField>, ParserError> {
```

Comment:
Can we include a comment linking to the duckdb docs where the struct syntax is defined?